engine/time/build.gradle

```diff
@@ -27,3 +27,9 @@ dependencies {
             project(path: ':test-configs')
     Classpaths.inheritSlf4j(project, 'slf4j-simple', 'testRuntimeOnly')
 }
+
+test {
+    jvmArgs '--add-exports', 'java.base/jdk.internal.misc=ALL-UNNAMED'
+    systemProperty 'RealTimeClockLoader.timeSource', 'jdk-internals'
```

Comment:
IMO, we should have an "escape" hatch, but the majority of cases shouldn't need to specify the system property IMO.